app/views/inbox/show.html.erb

```diff
@@ -64,8 +64,11 @@
                 </li>
               <% end %>
 
-              <li class="action">
-                <%= check_box_tag "inbox_comments[]", inbox_comment.id, false, :title => ts("select this comment"), :id => "inbox_comments_#{inbox_comment.id}" %>
+              <li>
+                <%= label_tag "inbox_comments_#{inbox_comment.id}" do %>
+                  <%= check_box_tag "inbox_comments[]", inbox_comment.id, false, :id => "inbox_comments_#{inbox_comment.id}" %>
```

Comment:
Minor style: can you use `id: ` instead of `:id => `?

Reply:
Absolutely! Resolved in 8f66cefa8.